### PR TITLE
[DO-1989] Preloader

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -11,14 +11,133 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logoDO192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link rel="stylesheet" href="%PUBLIC_URL%/css/app.min.css" />
+    <link rel="preload" href="%PUBLIC_URL%/css/app.min.css" as="style" onload="this.rel='stylesheet'" />
+    <script>
+      (function( w ){
```
Comment (roman-tiukh) [inline public/index.html:16]: this script should be moved to js file

File: public/index.html [hunk 2]
```diff
@@ -11,14 +11,133 @@
...
   </head>
   <body>
+    <div id="preloader"></div>
+
     <noscript>You need to enable JavaScript to run this app.</noscript>
 
     <div id="root"></div>
 
+    <script>
+      window.onload = (() => {
```
Comment (roman-tiukh) [inline public/index.html:136]: use addEventListener

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 2 above)
Comment (Iva-khar) [reply public/index.html:136]: done

File: public/index.html (hunk 1 above)
Comment (Iva-khar) [reply public/index.html]: done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**